docs/source/api/user-guide/measurements/canonical-noise-mechanism.ipynb

```diff
@@ -0,0 +1,408 @@
+    "\n",
+    "We also have a suite of inference tools that can be applied under the assumption that your statistic is binomially distributed.\n",
+    "An example of binomially-distributed data can be found when counting the number of married individuals in the California demographics dataset,\n",
+    "a microdata-level dataset with attributes for age, sex, education, race, income and marriage status."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import opendp.prelude as dp\n",
+    "import polars as pl\n",
+    "\n",
+    "dp.enable_features(\"contrib\")\n",
+    "columns = \"age\", \"sex\", \"educ\", \"race\", \"income\", \"married\"\n",
+    "married_data = list(pl.read_csv(\n",
+    "    dp.examples.get_california_pums_path(), \n",
+    "    has_header=False, \n",
+    "    new_columns=columns,\n",
+    "    ignore_errors=True,\n",
+    ")[\"married\"])\n",
+    "\n",
+    "context = dp.Context.compositor(\n",
+    "    data=married_data,\n",
+    "    privacy_unit=dp.unit_of(contributions=1),\n",
+    "    privacy_loss=dp.loss_of(epsilon=0.2, delta=1e-8),\n",
+    "    split_evenly_over=2\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll use the canonical noise mechanism to conduct statistical inference on the 'married' column,\n",
+    "where values are either 0 or 1, indicating marriage status.\n",
+    "\n",
+    "The following query counts the number of married individuals in the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = context.query().cast_default(float).clamp((0.0, 1.0)).sum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The canonical noise mechanism behaves similarly to the Laplace or Gaussian mechanism in that it is an additive noise mechanism.\n",
+    "\n",
+    "> The implementation of the canonical noise mechanism in OpenDP only supports scalar-valued float inputs and can only be calibrated to satisfy $(\\epsilon, \\delta)$-DP."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "546.4339998964982"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "estimate: float = query.canonical_noise().release()\n",
+    "estimate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Under $(\\epsilon, \\delta)$-DP,\n",
+    "the canonical noise distribution follows the Tulap distribution,\n",
```

Comment:
There's only a few remaining references to "Tulap" now: Double check that they all still belong.